st-thomas-more/stm-dashboard
# 3 add authentication

To test:

Follow instructions in README. Make sure you create the .env file. If you have trouble with the updated react-scripts stuff check out their [guide](https://github.com/facebookincubator/create-react-app/blob/master/packages/react-scripts/template/README.md#updating-to-new-releases) for updating to new releases. Start the app. Make sure you run it on port 3000 and visit it at localhost:3000. Then make sure you can login with a Google account. It should redirect you to the homepage after logging in and allow you to logout.
